src/utils/tryToLinkAutomatically.ts

```diff
@@ -0,0 +1,45 @@
+import { getEntityContactList, deleteEntityContact, setEntityContact } from "../services/entityAssociation";
+import { getContactsByEmailService } from "../services/hubspot";
+import { getEntityMetadata } from "../utils";
+import type { IDeskproClient } from "@deskpro/app-sdk";
+import type { DeskproUser, ContactDeps } from "../types";
+import type { Contact } from "../services/hubspot/types";
+
+const tryToLinkAutomatically = async (
+    client: IDeskproClient,
+    dpUser: DeskproUser,
+    getContactInfo: (contactId: Contact["id"]) => Promise<ContactDeps>,
+    linkContactFn: (contactId: Contact["id"]) => Promise<void>,
+) => {
+    try {
+        const contactIds = await getEntityContactList(client, dpUser.id);
+
+        if (contactIds?.length === 1) {
```

Comment:
Can you add comments on why these checks exist? I'm not too sure what's going on here.

Reply:
done

Reply:
![image](https://github.com/user-attachments/assets/74a4538d-f155-4425-9946-cad150c93f06)
I see no comments?